docs/source/ecf.rst

```diff
@@ -480,13 +480,17 @@ interp_method
 '''''''''''''
 Only used for photometry analyses. Interpolate bad pixels. Options: None (if no interpolation should be performed), linear, nearest, cubic
 
+centroid_method
+'''''''''''''''
+Only used for photometry analyses. Selects the method used for determining the centroid position (options: fgc or mgmc). For it's initial centroid guess, the 'mgmc' method creates a median frame from each batch of integrations and performs centroiding on the median frame (with the exact centroiding method set by the centroid_tech parameter). For each integration, the 'mgmc' method will then crop out an area around that guess using the value of ctr_cutout_size, and then perform a second round of centroiding to measure how the centroid moves over time. The 'fgc' method is the legacy centroiding method and is not currently recommended.
+
 ctr_guess
 '''''''''
 Optional, and only used for photometry analyses. An initial guess for the [x, y] location of the star that will replace the default behavior of first doing a full-frame Gaussian centroiding to get an initial guess.
 
 ctr_cutout_size
 '''''''''''''''
-Only used for photometry analyses. Amount of pixels all around the guessed centroid location which should be used for the more precise second centroid determination after the coarse centroid calculation. E.g., if ctr_cutout_size = 10 and the centroid (as determined after coarse step) is at (200, 200) then the cutout will have its corners at (190,190), (210,210), (190,210) and (210,190). The cutout therefore has the dimensions 21 x 21 with the centroid pixel (determined in the coarse centroiding step) in the middle of the cutout image.
+Only used for photometry analyses. For the 'fgc' method this parameter is the amount of pixels all around the guessed centroid location which should be used for the more precise second centroid determination after the coarse centroid calculation. E.g., if ctr_cutout_size = 10 and the centroid (as determined after coarse step) is at (200, 200) then the cutout will have its corners at (190,190), (210,210), (190,210) and (210,190). The cutout therefore has the dimensions 21 x 21 with the centroid pixel (determined in the coarse centroiding step) in the middle of the cutout image. For the 'mgmc' method it is the same however the value must be odd E.g. ctr_cutout_size = 11.
```

Comment:
Are you sure that it is the same - if so, there'd be absolutely no reason for it to have to be odd... If it really must be odd, then the MIRI ECF template will need to be updated (currently set to 10)

Reply:
Yes, I think it does need to be odd. ctr_cutout_size is used as the variable for box_size in th eentoid_sources function and in the documentation it states that "box_size must have odd values for both axes." I also tested it out in a jupyter notebook and it errors out if an even number is used stating: "ValueError: box_size must have an odd value for both axes". I updated the MIRI ECF to address it.

Reply:
If that's the case, then I really don't think this variable has the same meaning as in the fgc method

Reply:
The documentation below makes it seem like `box_size` is the *full-width* of the cutout box, while fgc treats `2*ctr_cutout_size+1` as the full-width.

<img width="601" alt="Screen Shot 2023-04-13 at 09 55 08" src="https://user-images.githubusercontent.com/24514122/231830826-9b3604ab-a146-4f0c-af11-977c6ab68a1b.png">

Reply:
How does this sound? "For the 'mgmc' method ctr_cutout_size is the full length of the cutout image along each axis and has to be an odd value. E.g. ctr_cutout_size = 11 then the cutout image will have the dimensions of 11 x 11 with the centroid pixle (determined in the first centroid guess) in the middle of the cutout image."

Reply:
Maybe to keep things simple we can keep the original meaning of `ctr_cutout_size` and do the conversion inside the code? Aka, pass `box_size=(2*ctr_cutout_size+1)` into centroid_sources? And then adjust the NIRCam template to use a value of 5 instead and leave MIRI at a value of 10

Reply:
Sure, I'll update the code.

Reply:
How is this? "Only used for photometry analyses. For the 'fgc' method this parameter is the amount of pixels all around the guessed centroid location which should be used for the more precise second centroid determination after the coarse centroid calculation. E.g., if ctr_cutout_size = 10 and the centroid (as determined after coarse step) is at (200, 200) then the cutout will have its corners at (190,190), (210,210), (190,210) and (210,190). The cutout therefore has the dimensions 21 x 21 with the centroid pixel (determined in the coarse centroiding step) in the middle of the cutout image. For the 'mgmc' method ctr_cutout_size is the full length of the cutout image along each axis. E.g. , if ctr_cutout_size = 10 then the cutout image will be (2*ctr_cutout_size+1) along the x-axis and (2*ctr_cutout_size+1) along the y-axis, giving the cutout image the dimensions of 21 x 21 with the first centroid guess pixle in the center."  

I also noticed that in the 'fgc' defintion it gives the dimensions of "(190,190), (210,210), (190,210) and (210,190). The cutout therefore has the dimensions 21 x 21" but 210-190 = 20 would that not give a dimension of 20 x 20?